Sources/MessagingPush/MessagingPushImplementation.swift

```diff
@@ -50,24 +38,15 @@ class MessagingPushImplementation: MessagingPushInstance {
     }
 
     func deleteDeviceToken() {
-        // FIXME: [CDP] Pass to Journey
-        // customerIO?.deleteDeviceToken()
-
-        eventHandlingManager.sendOrSaveEvent(event: DeleteDeviceTokenEvent())
+        eventHandlingManager.postEvent(DeleteDeviceTokenEvent())
     }
 
     func registerDeviceToken(_ deviceToken: String) {
-        // FIXME: [CDP] Pass to Journey
-        // customerIO?.registerDeviceToken(deviceToken)
-
-        eventHandlingManager.sendOrSaveEvent(event: RegisterDeviceTokenEvent(token: deviceToken))
+        eventHandlingManager.postEvent(RegisterDeviceTokenEvent(token: deviceToken))
     }
 
     func trackMetric(deliveryID: String, event: Metric, deviceToken: String) {
-        // FIXME: [CDP] Pass to Journey
-        // customerIO?.trackMetric(deliveryID: deliveryID, event: event, deviceToken: deviceToken)
-
-        eventHandlingManager.sendOrSaveEvent(event: TrackMetricEvent(deliveryID: deliveryID, event: event.rawValue, deviceToken: deviceToken))
+        eventHandlingManager.postEvent(TrackMetricEvent(deliveryID: deliveryID, event: event.rawValue, deviceToken: deviceToken))
```

Comment:
```suggestion
        eventHandlingManager.postEvent(TrackMetricEvent(deliveryID: deliveryID, event: event, deviceToken: deviceToken))
```

Similar comment about not needing to use `.rawValue`.